<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NyxorADCWidget</class>
 <widget class="QWidget" name="NyxorADCWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="PhasegroupBox">
     <property name="title">
      <string>DC0 Phase</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLineEdit" name="DC0PhaselineEdit">
        <property name="toolTip">
         <string>DC0 Phase register</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="TransmitgroupBox">
     <property name="title">
      <string>Transmit pattern</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="Transmit1LineEdit">
        <property name="toolTip">
         <string>Transmit pattern register 1</string>
        </property>
        <property name="inputMask">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="Transmit2LineEdit">
        <property name="toolTip">
         <string>Transmit pattern register 2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="Transmit3LineEdit">
        <property name="toolTip">
         <string>Transmit pattern register 3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="Transmit4LineEdit">
        <property name="toolTip">
         <string>Transmit pattern register 4</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>89</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
